samples/maps/series/latlon-transform/demo.js

```diff
@@ -3,7 +3,7 @@ var chart,
     position;
 
 // Initiate the chart
-chart = Highcharts.Map({
+chart = new Highcharts.Map({
```

Comment:
I believe that this requires, that `Highcharts.Map = Highcharts.MapChart.mapChart`.

Reply:
Sorry, to be more specific: IMHO we should not change this test.

Reply:
Since this is example code intended for people to look at, maybe it should just use `mapChart`?

Reply:
Agreed. Maybe you can find some other test for `Highcharts.Map` without instantiation.

Reply:
Yeah I added one to the constructor unit test.